sh/tests/json-researcher_test.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# Unofficial Bash Strict Mode
+set -euo pipefail
+IFS='
+'
+
+echo insecure >> ~/.curlrc
+caddy start &>/dev/null
+
```

Comment:
If the `id` is not given in the subject - here there should be a line to pick a random number between the possible heroes ids to pass to the 2 scripts

Reply:
the goal from this exercise is not to teach the student about the script arguments, the script arguments will be teached in the shell scripting quests